front/src/app/components/forms/convention/ConventionSummary.tsx

```diff
@@ -110,6 +110,12 @@ const beneficiarySummary = (convention: ConventionReadDto) => {
       fields["signatories.beneficiary.birthdate"].label,
       convention.signatories.beneficiary.birthdate,
     ],
+    convention.internshipKind === "mini-stage-cci"
+      ? [
+          fields["signatories.beneficiary.levelOfEducation"].label,
+          convention.signatories.beneficiary.levelOfEducation,
+        ]
+      : ["", ""],
```

Comment:
je te propose plutôt :

```suggestion
    ...(convention.internshipKind === "mini-stage-cci"
      ? [[
          fields["signatories.beneficiary.levelOfEducation"].label,
          convention.signatories.beneficiary.levelOfEducation,
        ]]
      : [],
```

Reply:
c'est ce que j'avais essayé au début mais mais passe pas niveau typage le tableau vide